assets/stylesheets/_sections.scss

```diff
@@ -57,13 +68,18 @@
         linear-gradient(rgba(black, 0.5), rgba(black, 0.5)),
         black
         url("#{$asset-path}" + "#{$joyce-background}");
-  background-position: center bottom;
+  background-position: bottom;
   background-repeat: no-repeat;
-  background-size: auto 150%;
+  @media screen and (min-width: $bp-small){
```

Comment:
what's `$bp`?

Reply:
oh, breakpoint. derp.

Reply:
derrrrpy (i totally lifted that variable name from the internet)